vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -40,4 +40,10 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_groups
+    num_of_container_groups = @record.number_of(:container_groups)
+    label = "Container groups"
+    {:label => label, :image => "container_group", :value => num_of_container_groups}
```

Comment:
> @abonas can you change the newly added strings to use gettext there are some examples in other textual helpers, this one can be changed to ui_lookup(:tables => "container_group") or _('container groups'), can you change others in this PR and in #2661

ok, I understand regarding ui_lookup, but can you point to an example for gettext? I went over several other helpers and found none.